ios/RCTVideo.m

```diff
@@ -686,7 +686,8 @@ - (void)setSeek:(NSDictionary *)info
     CMTime cmSeekTime = CMTimeMakeWithSeconds([seekTime floatValue], timeScale);
     CMTime current = item.currentTime;
     // TODO figure out a good tolerance level
-    CMTime tolerance = CMTimeMake([seekTolerance floatValue], timeScale);
+    CMTime tolerance = CMTi
```

Comment:
Doubt

Reply:
Not sure why, but this is shown correctly within the code editor. I think it's a bug in Github.